.github/workflows/slack-pr-notifications.yaml

```
@@ -0,0 +1,754 @@
+
+  notify-pr-review:
+    name: Send PR Review Notification
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request_review'
+    permissions:
+      contents: read
+      pull-requests: read
+    steps:
+      - name: Get review details
+        id: review-details
+        uses: actions/github-script@v6
```

actionlint (1.7.9)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)